e2e/firstTest.spec.js

```diff
@@ -7,22 +7,24 @@ describe("Main screen", () => {
   });
 
   /**
-  * The following source code represents the following automated test scenario
-  * 1. Load the app
-  * 2. Display main screen
-  * 3. Click on the searchbar
-  * 4. Type the query: "hall building"
-  * 5. Select the corresponding item from the dropdown list
-  * 6. Click on the bottom menu (not yet implemented)
-  * 7. Scroll down and read the information provided (not yet implemented)
-  */
+   * The following source code represents the following automated test scenario
+   * 1. Load the app
+   * 2. Display main screen
+   * 3. Click on the searchbar
+   * 4. Type the query: "hall building"
+   * 5. Select the corresponding item from the dropdown list
+   * 6. Click on the bottom menu (not yet implemented)
+   * 7. Scroll down and read the information provided (not yet implemented)
+   */
 
-  it.skip("Should click on the searchbar and type: hall building", async () => {
+  it("Should click on the searchbar and type: hall building", async () => {
     await element(by.id("search")).tap();
     await element(by.id("search")).typeText("hall building");
     await element(by.id("search")).tapAtPoint({ x: 200, y: 75 });
 
     // TODO: Fix x and y coordinates so that it clicks on the BottomMenu
     // await element(by.id("search")).tapAtPoint({ x: 50, y: 890 });
+    await expect(element(by.id("iconBottom2"))).toExist();
+    await element(by.id("iconBottom2")).tapAtPoint({ x: 30, y: 40 });
```

Comment:
This refers to the 2 comments in BottomMenu:
1- the naming of TestIDs
2- We need to find a real fix at some point instead of the workaround for clicking a button inside a view